packages/example-rpc-server/index.js

```diff
@@ -7,5 +7,8 @@ const application = (module.exports = require('./dist'));
 
 if (require.main === module) {
   // Run the application
-  application.main();
+  application.main().catch(err => {
+    console.log('Cannot start the application.', err);
```

Comment:
`console.error`

Reply:
Do you want the same change in `example-hello-world`?

Reply:
Yes please. All examples should be consistent and leverage this approach.